test/make_dist.py

```diff
@@ -51,19 +50,16 @@ def build_dist():
 
 
 def download_cache(wait=False):
-    tries = 50 if wait else 1  # 25 minutes, once every 30s
-    for retry in range(tries):
-        try:
-            subprocess.check_call(["tools/webpack-jumpstart"])
-            return True
-        except subprocess.CalledProcessError as e:
-            if e.returncode != 2 or not wait:
-                break
-            message("make_dist: pre-built dist not yet available, waiting...")
-            time.sleep(30)
-
-    message("make_dist: Downloading pre-built dist failed")
-    return False
+    cmd = 'tools/webpack-jumpstart'
```

Comment:
This needs to be an array, otherwise the `.append()` below will crash. check_call() may get along with a single string (maybe implying shell, but hopefully assuming just a single argument-less exec name).

Reply:
thanks!

i wrote this section twice, and as usual, had it right in the first time, and messed it up in the rewrite :)